en-GB.lproj/Main.strings

```diff
@@ -192,46 +192,46 @@
 
 /* Class = "UITextField"; placeholder = "365 Days"; ObjectID = "Zx2-Kx-0aV"; */
-"Zx2-Kx-0aV.placeholder" = "365 Days";
+"Zx2-Kx-0aV.placeholder" = "365 Dagen";
 
 /* Class = "UILabel"; text = "Auto respring"; ObjectID = "b6w-j2-Rh6"; */
-"b6w-j2-Rh6.text" = "Auto respring";
+"b6w-j2-Rh6.text" = "Automatische respring";
 
 /* Class = "UILabel"; text = "Load Daemons"; ObjectID = "bBi-OR-hpE"; */
-"bBi-OR-hpE.text" = "Load Daemons";
+"bBi-OR-hpE.text" = "laad Daemons";
 
 /* Class = "UITextView"; text = "[*] unc0ver Version: 3.1.3+7.gf5833f2\n[*] Machine Name: iPhone10,5\n[*] Model Name: D211AP\n[*] Kernel Version: Darwin Kernel Version 18.7.0: Tue May 21 01:53:36 PDT 2019; root:xnu-4903.270.29~10/RELEASE_ARM64_T8015\n[*] Kernel Page Size: 0x4000\n[*] System Version: iOS 12.4 (Beta) (Build: 16G5038d)\n[*] Bundled Resources Version: 1.0~b21\n[*] Status: Unsupported\n[*] Progress: Unsupported\n[MC] System group container for systemgroup.com.apple.configurationprofiles path is /private/var/containers/Shared/SystemGroup/systemgroup.com.apple.configurationprofiles\n[MC] Reading from public effective user settings.\n"; ObjectID = "bvv-3a-l9y"; */
 "bvv-3a-l9y.text" = "[*] unc0ver Version: 3.1.3+7.gf5833f2\n[*] Machine Name: iPhone10,5\n[*] Model Name: D211AP\n[*] Kernel Version: Darwin Kernel Version 18.7.0: Tue May 21 01:53:36 PDT 2019; root:xnu-4903.270.29~10/RELEASE_ARM64_T8015\n[*] Kernel Page Size: 0x4000\n[*] System Version: iOS 12.4 (Beta) (Build: 16G5038d)\n[*] Bundled Resources Version: 1.0~b21\n[*] Status: Unsupported\n[*] Progress: Unsupported\n[MC] System group container for systemgroup.com.apple.configurationprofiles path is /private/var/containers/Shared/SystemGroup/systemgroup.com.apple.configurationprofiles\n[MC] Reading from public effective user settings.\n";
 
 /* Class = "UIButton"; normalTitle = "@MiscMisty (Development)"; ObjectID = "cUa-LX-1Nb"; */
-"cUa-LX-1Nb.normalTitle" = "@MiscMisty (Development)";
+"cUa-LX-1Nb.normalTitle" = "@MiscMisty (Ontwikkelaar)";
 
 /* Class = "UIButton"; normalTitle = "@realBrightiup (Development)"; ObjectID = "egd-uq-xg0"; */
-"egd-uq-xg0.normalTitle" = "@realBrightiup (Development)";
+"egd-uq-xg0.normalTitle" = "@realBrightiup (Ontwikkelaar)";
 
 /* Class = "UILabel"; text = "Reset Cydia Cache"; ObjectID = "fyE-Sj-TNL"; */
-"fyE-Sj-TNL.text" = "Reset Cydia Cache";
+"fyE-Sj-TNL.text" = "herstel Cydia Cache";
 
 /* Class = "UIButton"; normalTitle = "@i41nbeer (Development)"; ObjectID = "gQp-2a-3sA"; */
-"gQp-2a-3sA.normalTitle" = "@i41nbeer (Development)";
+"gQp-2a-3sA.normalTitle" = "@i41nbeer (Ontwikkelaar)";
 
 /* Class = "UIButton"; normalTitle = "@_Matchstic (Development)"; ObjectID = "gSs-Qr-0L6"; */
-"gSs-Qr-0L6.normalTitle" = "@_Matchstic (Development)";
+"gSs-Qr-0L6.normalTitle" = "@_Matchstic (Ontwikkelaar)";
 
 /* Class = "UILabel"; text = "Dump APTicket"; ObjectID = "gi9-pl-mUW"; */
-"gi9-pl-mUW.text" = "Dump APTicket";
+"gi9-pl-mUW.text" = "Stortplaats APTicket";
```

Comment:
"Dump" is a valid Dutch word so this sentence doesn't need translation. "Stortplaats"? I'm almost certain you used Google Translate for this?